src/Flicking.ts

```diff
@@ -123,13 +123,11 @@ class Flicking extends Component {
     const currentPanel = this.getCurrentPanel();
     const currentState = this.viewport.stateMachine.getState();
 
-    if (!currentPanel || currentState.type !== STATE_TYPE.IDLE) {
-      return this;
```

Comment:
@WoodNeck I wonder why do you think it makes complexity?

Reply:
I'm currently using [CodeMetrics](https://marketplace.visualstudio.com/items?itemName=kisstkondoros.vscode-codemetrics) for practicing purpose.
They regard `return` phrase makes more complexity, so I've refactored it.